NameLayerMain/src/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -222,6 +223,14 @@ public void run() {
 			ver = updateVersion(ver, plugin.getName());
 			log(Level.INFO, "Database update to Version seven took " + (System.currentTimeMillis() - first_time) /1000 + " seconds.");
 		}
+		if (ver == 7){
+			long first_time = System.currentTimeMillis();
+			log(Level.INFO, "Database updating to version eight.");
+			db.execute("alter table faction add last_timestamp datetime NOT NULL default NOW();");
+			ver = updateVersion(ver, plugin.getName());
+			log(Level.INFO, "Database update to Version seven took " + (System.currentTimeMillis() - first_time) /1000 + " seconds.");
```

Comment:
Sanity in reading logging -- change this line to match line 228, "Version eight" instead of seven :)